Framework/PythonInterface/mantid/plots/plotfunctions.py

```diff
@@ -357,9 +359,176 @@ def pcolormesh(axes, workspace, *args, **kwargs):
 
     return axes.pcolormesh(x, y, z, *args, **kwargs)
 
+def _imshow(axes, X, cmap=None, norm=None, aspect=None,
+           interpolation=None, alpha=None, vmin=None, vmax=None,
+           origin=None, extent=None, shape=None, filternorm=1,
+           filterrad=4.0, imlim=None, resample=None, url=None, **kwargs):
+    """
```

Comment:
Change the documentation to be a reference to the matplotlib docs (like you have at the bottom) and the differences.